From fd8e3305f7696b6f6753e3164ea8f8f121d523a0 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 25 Jul 2020 10:59:33 -0400
Subject: [PATCH 1/4] add NearbyMessagingService & DiscoveryService

  from
  https://github.com/LineageOS/android_device_oneplus_oneplus2/commit/ 9b916939432bb5dda339bb2405f4c45ca82f80f4#diff-e0a3646285a5f4f89d0d2f1cd05c0458
  https://github.com/LineageOS/android_device_oneplus_oneplus2/commit/ 05331348b64539fbd78f898dee00300c6cc02a50#diff-e0a3646285a5f4f89d0d2f1cd05c0458

Change-Id: Idf0200a0b9ef2199b85e02bf0d2a5ccd2a081dc3
---
 overlay/common/frameworks/base/core/res/res/values/config.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/overlay/common/frameworks/base/core/res/res/values/config.xml b/overlay/common/frameworks/base/core/res/res/values/config.xml
index 1f21e868..2cc908e9 100644
--- a/overlay/common/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/common/frameworks/base/core/res/res/values/config.xml
@@ -77,6 +77,8 @@
         <item>com.google.android.gms/com.google.android.gms.update.UpdateFromSdCardService</item>
         <item>com.google.android.gms/com.google.android.gms.update.phone.PopupDialog</item>
         <item>com.google.android.gms/com.google.android.gms.update.resumeonreboot.ResumeOnRebootEscrowService</item>
+        <item>com.google.android.gms/com.google.android.gms.nearby.messages.service.NearbyMessagesService</item>
+        <item>com.google.android.gms/com.google.android.gms.nearby.discovery.service.DiscoveryService</item>
     </string-array>
 
     <!-- Enable lockscreen rotation by default -->
-- 
2.25.1

